website/docs/r/build_definition.html.markdown

```diff
@@ -11,6 +11,7 @@ Manages a Build Definition within Azure DevOps.
 
 ## Example Usage
 
+### Tfs
```

Comment:
what does `Tfs` mean ?

Reply:
Team Foundation Server + Git. It's an example when `repo_type` is `TfsGit`